.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,41 @@
+name: Unit Tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    name: Run unit tests in containers
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        scenarios:
+          - name: Run unit tests with python3.9 on el8
+            python: python3.9
+            container: ubi8
+          - name: Run unit tests with python 3.6 on el8
+            python: python3.6
+            container: ubi8
+          - name: Run unit tests with python2.7 on el7
+            python: python2.7
+            container: ubi7
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          # NOTE(ivasilev) fetch-depth 0 is critical here as leapp deps discovery depends on specific substring in
```

Comment:
This comment can be removed as we don't have leapp-repository PR discovery magic in Makefile for leapp